pkg/data/management/kontainerdriver_data.go

```diff
@@ -101,6 +101,17 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 		return err
 	}
 
+	if err := creator.addCustomDriver(
+		"otccce",
```

Comment:
```suggestion
		"opentelekomcloudcloudcontainerengine",
```

Reply:
I suppose I will change it to `opentelekomcloudcontainerengine`, same as above drivers